crates/pipeline_manager/migrations/V11__service_probe.sql

```diff
@@ -0,0 +1,12 @@
+
+CREATE TABLE IF NOT EXISTS service_probe (
```

Comment:
Let's say the service_probe "pings" a remote service. If it pings it 10 times, are these 10 rows in this table? Is there any value to these rows being in the DB indefinitely?

I'd say it's worth having a `probe_type` field too, even if it is just a placeholder for now. This is in case we want to have different types of probes for a given service.

Reply:
New version is only active rather than continuous in background.
There is a `request_type` field now.